packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -562,22 +567,203 @@ You can check the status of your Validator, including whether it is elected and
 
 ```bash
 # On your local machine
-celocli validator:status --validator $CELO_VALIDATOR_RG_ADDRESS
+celocli validator:status --validator $CELO_VALIDATOR_ADDRESS
 ```
 
 You can see additional information about your Validator, including uptime score, by running:
 
 ```bash
 # On your local machine
-celocli validator:show $CELO_VALIDATOR_RG_ADDRESS
+celocli validator:show $CELO_VALIDATOR_ADDRESS
 ```
 
 ### Running the Attestation Service
 
-{% hint style="warning" %}
-**Under construction** We are currently working on updating the attestation service guide to reflect changes in workflow for `ReleaseGold`.
+As part of the [lightweight identity protocol](/celo-codebase/protocol/identity), Validators are expected to run an [Attestation Service](https://github.com/celo-org/celo-monorepo/tree/master/packages/attestation-service) to provide attestations that allow users to map their phone number to an account on Celo. Be sure to allow TCP connections to your Attestations machine on port 80 for all IP addresses.
+
+Just like with the Validator and Vote signer, we'll want to authorize a separate Attestation signer. For that let's start our node on the Attestations machine (keep track of the password you use for this account):
+
+{% hint style="info" %}
+An earlier version of this document set the beneficiary addresses to `CELO_VALIDATOR_ADDRESS` and `CELO_VALIDATOR_GROUP_ADDRESS`, but the attestation service expects these to refer to the `ReleaseGold` contract, so please move your beneficiary addresses to reflect the environment variables above.
```

Comment:
Does any of the attestations code actually read env vars? Or is this just an artifact of the original instructions? would it be easier to change those instructions to instead use the RG env var names?

Reply:
Unless the attestations service *code* specifically requires these env vars to be set, we should just change the attestations service instructions to use the RG env vars

Reply:
Some parts definitely do, although they may just be tests https://github.com/celo-org/celo-monorepo/blob/581939a2a6413cb74f6ba8b19b2b2e326590c562/packages/attestation-service/src/env.ts#L34

Was advised by @nambrot that it would be advised to not change in attestation (which is what I originally tried). I think it should be changed in the docs here as currently reflected, but I don't have a great grasp on the impact to validators

Reply:
As @lucasege, I don't think the attestation service should need to know that RG env vars exist